### ISSUE TITLE
10.3.6: 415 unsupported media type via DLNA

### ISSUE THREAD

**Issue by nonplusnl** [jellyfin/jellyfin#1532]
In 10.3.6, I can no longer play to UpMPD DLNA clients, Jellyfin throws error (see below).
Downgraded to 10.3.5 and everything works fine

To reproduce:
* Select UpMPD DLNA client in Jellyfin web
* Select audio to play, press play
* Nothing happens

Expected behaviour
* Playback via selected DLNA client

Log entries:
```
[2019-07-10 09:17:53.751 +02:00] [ERR] HTTP request failed with message: "<html><body><h1>415 Unsupported Media Type</h1></body></html>"
[2019-07-10 09:17:53.755 +02:00] [ERR] Error updating device info for "Keuken"
MediaBrowser.Model.Net.HttpException: Unsupported Media Type
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.EnsureSuccessStatusCode(HttpResponseMessage response, HttpRequestOptions options)
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.SendAsyncInternal(HttpRequestOptions options, HttpMethod httpMethod)
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.SendAsync(HttpRequestOptions options, HttpMethod httpMethod)
   at Emby.Dlna.PlayTo.SsdpHttpClient.SendCommandAsync(String baseUrl, DeviceService service, String command, String postData, Boolean logRequest, String header)
   at Emby.Dlna.PlayTo.Device.GetTransportInfo(TransportCommands avCommands, CancellationToken cancellationToken)
   at Emby.Dlna.PlayTo.Device.TimerCallback(Object sender)
```

...and...

```
[2019-07-10 09:41:55.643 +02:00] [ERR] HTTP request failed with message: "<html><body><h1>415 Unsupported Media Type</h1></body></html>"
[2019-07-10 09:41:55.647 +02:00] [ERR] Error processing request
MediaBrowser.Model.Net.HttpException: Unsupported Media Type
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.EnsureSuccessStatusCode(HttpResponseMessage response, HttpRequestOptions options)
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.SendAsyncInternal(HttpRequestOptions options, HttpMethod httpMethod)
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.SendAsync(HttpRequestOptions options, HttpMethod httpMethod)
   at Emby.Dlna.PlayTo.SsdpHttpClient.SendCommandAsync(String baseUrl, DeviceService service, String command, String postData, Boolean logRequest, String header)
   at Emby.Dlna.PlayTo.Device.SetAvTransport(String url, String header, String metaData, CancellationToken cancellationToken)
   at Emby.Dlna.PlayTo.PlayToController.SetPlaylistIndex(Int32 index)
   at Emby.Dlna.PlayTo.PlayToController.PlayItems(IEnumerable`1 items)
   at Emby.Dlna.PlayTo.PlayToController.SendPlayCommand(PlayRequest command, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Session.SessionManager.SendMessageToSession[T](SessionInfo session, String name, T data, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Session.SessionManager.SendPlayCommand(String controllingSessionId, String sessionId, PlayRequest command, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Services.ServiceExecGeneral.GetTaskResult(Task task)
   at Emby.Server.Implementations.Services.ServiceHandler.ProcessRequestAsync(HttpListenerHost appHost, IRequest httpReq, IResponse httpRes, ILogger logger, String operationName, CancellationToken cancellationToken)
   at Emby.Server.Implementations.HttpServer.HttpListenerHost.RequestHandler(IHttpRequest httpReq, String urlString, String host, String localPath, CancellationToken cancellationToken)
```

System
- Ubuntu 18.04, package-installed (not Docker)
- Browsing via Edgium, playback via DLNA -> UpMPD -> MPD. These run on server
- Jellyfin 10.3.6 (bug introduced in 10.3.6; downgrading to 10.3.5. "solves" issue)
- Reverse proxy with https, Apache. I have set Jellyfin "Secure connection mode" to disabled, to solve other connection issues when using reverse proxy

**Comment by Bond-009** [comment]
#1522 should fix this

**Comment by anthonylavado** [comment]
@nonplusnl We've recently released 10.3.7, with the mentioned fixes. Can you test again and let us know if it's still an issue? Thanks!

**Comment by nonplusnl** [comment]
Yes sir, issue solved in 10.3.7, thanks!
